bootstrap

```diff
@@ -629,7 +629,7 @@ cd ..
 
 # Add -I for in-tree macro directory first, so we use that xapian.m4 in
 # preference to one in any directories which may be added below.
-ACLOCAL="${ACLOCAL-aclocal} -I `pwd`/xapian-core/m4-macros"
+ACLOCAL="${ACLOCAL-aclocal} -I `pwd`/xapian-core/m4-macros -I `pwd`/xapian-letor/m4-macros"
```

Comment:
We need to add xapian-core's `m4-macros` so omega, bindings and letor can find `xapian.m4`.  We don't need anything else to find letor's macro (though I guess that might change if bindings for letor are added).

Reply:
@olly I am keeping this change as it is for now as letor bindings are going to be added soon.

Reply:
I'm not `@olly`...

But that seems OK to me.